configure.ac

```diff
@@ -28,11 +28,21 @@ AX_CHECK_COMPILE_FLAG([-fsanitize=undefined], [UBSAN_CFLAGS=-fsanitize=undefined
 AC_SUBST(UBSAN_CFLAGS)
 
 AC_ARG_ENABLE(debug,
-	      AS_HELP_STRING([--enable-debug=@<:@yes/no@:>@],
-			     [Enable extra debugging information for font
-			      sanitization @<:@default=no@:>@]),
-	      AC_DEFINE(OTS_DEBUG, 1, [Enable extra debugging information]),
-	      :)
+              AS_HELP_STRING([--enable-debug=@<:@yes/no@:>@],
+              [Enable extra debugging information for font
+               sanitization @<:@default=no@:>@]),
+              if test $enableval = yes; then
+                AC_DEFINE(OTS_DEBUG, 1, [Enable extra debugging information])
+              fi,
+              :)
```

Comment:
Okay. In particular, the lines you have highlighted would change anyway because I wrapped the AC_DEFINE in an if statement.